drafts/0007-RFC-BAG-OPERATORS.md

```diff
@@ -4,26 +4,26 @@
 # Summary
 [summary]: #summary
 
-This RFC defines the PartiQL specification for the operators: UNION, INTERSECT, and EXCEPT.
+This RFC defines the PartiQL specification for the operators: UNION, INTERSECT, and EXCEPT. The behavior of each operator conforms to the SQL standard in strict typing mode. In the absence of schemas, each operator is equivalent to the mathematical definitions of the respective multi-set operator where member equality is consistent with the equality in the *GROUP BY* clause.
```

Comment:
I believe irrespective of _strict_ vs _permissive_ mode, the behavior conforms.  Put another way, if we were in _permissive_ mode and there was schema, and a `UNION` operator, the behavior should be SQL compatible.

Reply:
As discussed in the review meeting, UNION, INTERSECT, and EXCEPT will conform to the SQL specification in all cases, and the pure mathematical union etc. will be separate operators named OUTER UNION, OUTER INTERSECT, and OUTER EXCEPT